capa/ida/plugin/form.py

```diff
@@ -652,13 +655,14 @@ def slot_progress_feature_extraction(text):
 
             update_wait_box("loading rules")
 
-            # function should handle exceptions and return False
-            if not self.load_capa_rules():
+            # load_capa_rules should always return
+            self.program_analysis_ruleset_cache = self.load_capa_rules()
```

Comment:
it does return, I think the comment can go and maybe we check/ensure via mypy/type annotation